src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,12 @@
+@mixin default-text {
+  font-family: "Roboto", Helvetica, sans-serif;
```

Comment:
We have this font-family on all our page. I'd add this line not to mixin, but to tag html